client/src/components/layout/TopBar.js

```diff
@@ -15,6 +22,11 @@ const TopBar = ({ user }) => {
   ];
 
   const authenticatedListItems = [
+    <li key="username">
+      <button type="button" className="button" >
+        Username: {username}
+      </button>
```

Comment:
I believe you can add the className `button` to the `li` element to make it look like a button. The `button` element shouldn't be used because there is no `click` functionality attached